### DISCUSSION TITLE
v24 docker image?

### DISCUSSION THREAD

**Discussion by Blackbat007** [nzbgetcom/nzbget#252] [General]
NZBget v23 is telling me there's a stable v24 release available but when I try and pull nzbgetcom/nzbget:latest I'm still getting v23. Will this be updated soon? The link to the release notes is still pointing to v23 too.

Great work on taking this project forwards - please keep it up!

**Comment by phnzb** [comment]
@Blackbat007 v24 docker image pushed to DockerHub and ghcr.io. Feel free to get updated.
